k6/seed-data.sql

```
@@ -0,0 +1,170 @@
+-- 7. 댓글 약 30000건
+WITH review_ids AS (
+    SELECT id FROM reviews WHERE deleted_at IS NULL LIMIT 50000
+),
+user_ids AS (
+    SELECT id FROM users WHERE deleted_at IS NULL LIMIT 1000
+)
+INSERT INTO comments (id, review_id, user_id, content, created_at, updated_at)
+SELECT
+    gen_random_uuid(),
+    r.id,
+    (SELECT id FROM user_ids ORDER BY random() LIMIT 1),
```

SQLFluff (4.1.0)

[error] The 'ORDER' keyword should always start a new line.

(LT14)